src/main/java/uk/gov/companieshouse/api/strikeoffobjections/processor/ObjectionProcessor.java

```diff
@@ -70,7 +75,7 @@ public void process(Objection objection, String httpRequestId)
 
         // TODO update status to PROCESSING
 
-        // TODO OBJ-139/OBJ-20 do chips sending
+        chipsService.sendObjection(httpRequestId, objection);
```

Comment:
I'd like to catch any exceptions thrown from chips service so we can log them, don't want to have to rely on other classes to do the logging. We can re-throw the exception (I know there's a log/throw anti pattern but I'd like to make sure nothing slips through).

Reply:
Ok. Will implement as we've done for the e-mail calls.